src/workerd/api/BUILD.bazel

```diff
@@ -79,6 +79,7 @@ wd_cc_library(
         "//src/pyodide",
         "//src/pyodide:pyodide_extra_capnp",
         "//src/workerd/api/node",
+        "//src/workerd/api/pyodide",
```

Comment:
This wasn't listed here so far. I was wondering why the build was working so far despite that – turns out that we were failing to exclude pyodide/pyodide.h from hdrs so far. With the pyodide folder now being a subpackage based on adding a BUILD.bazel file there, the file won't be captured by the glob so this is no longer an issue.